src/ts/components/subtitleoverlay.ts

```diff
@@ -13,8 +14,17 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
   private subtitleManager: ActiveSubtitleManager;
   private previewSubtitleActive: boolean;
   private previewSubtitle: SubtitleLabel;
+  private cea608lineHeight: number;
+  private isCEA608: boolean = false;
 
   private static readonly CLASS_CONTROLBAR_VISIBLE = 'controlbar-visible';
+  private static readonly CLASS_CEA_608 = 'cea608';
+  private static readonly CEA608_NUM_COLUMN = 32;
+  private static readonly CEA608_WIDTH = 0.8;
```

Comment:
Where are the values of all these constants coming from? Please add comments (similar to `CEA608_LINE_COEF `) so others can understand why they have these values.